doc/man1/qsub.1B

```diff
@@ -1437,9 +1445,11 @@ is sent to the job submitter stating the error.
 
 Dependency examples:
 .RS 3
-.B "qsub -W depend=afterok:123.host1.domain.com /tmp/script"
+.B "job1=`qsub first_step.sh`"
 .br
-.B "qsub -W depend=before:234.host1.com:235.host1.com /tmp/script"
+.B "qsub -W depend=afterok:$job1 second_step.sh"
+.sp
+.B "qsub -W depend=before:234.host1.com:235.host1.com jobscript"
```

Comment:
Did you mean to accept the output of the second qsub into $job2 and then use $job1 and $job2 here?

Reply:
The two examples were intended to be independent. In looking at them more, I do not like the before example and will remove it entirely.